### PR TITLE
Deploy smoker on EL8 to support client testing

### PR DESCRIPTION
**Pull request by ehelms**

### CODE REVIEW

File: roles/smoker/defaults/main.yml [hunk 1]
```diff
@@ -9,6 +9,5 @@ smoker_command_args: "--driver chrome --variables '{{ smoker_variables_path }}'
 smoker_markers:
 # pinned to 86 as 87 hangs randomly: https://bugzilla.redhat.com/show_bug.cgi?id=1914166
 smoker_browser_packages:
-  - https://people.redhat.com/evgeni/chrome86/chromedriver-86.0.4240.193-1.el7.x86_64.rpm
-  - https://people.redhat.com/evgeni/chrome86/chromium-86.0.4240.193-1.el7.x86_64.rpm
-  - https://people.redhat.com/evgeni/chrome86/chromium-common-86.0.4240.193-1.el7.x86_64.rpm
+  - chromedriver
+  - chromium
```
Comment (ehelms) [inline roles/smoker/defaults/main.yml:13]: @evgeni Is there EL8 versions of the above you be up for hosting in the same way? I think I am seeing the same timeout issue using these from the repositories.

Comment (ekohl) [reply roles/smoker/defaults/main.yml:13]: I think we pulled the old version from Fedora's koji so I'd search there. However, https://koji.fedoraproject.org/koji/buildinfo?buildID=1641050 was deleted.

Comment (evgeni) [reply roles/smoker/defaults/main.yml:13]: Nope. Best thing I can offer is to pull the source from Fedora's dist git and throw it at a friendly koji that knows EL8 ;)

**Comment by ekohl** [comment]
Can we revisit this now that we're dropping EL7?

**Comment by evgeni** [comment]
> Can we revisit this now that we're dropping EL7?

Sure. I've rebased, fixed the conflicts and moved to c8s instead of c8, but didn't actually test any of this yet.

**Review by ekohl — APPROVED** [review]
I suppose we have one way to find it.

**Comment by evgeni** [comment]
Well, this passed locally, so going to merge!

**Pull request merged**